src/main/resources/schemas/nhinc/common/configAdmin.xsd

```diff
@@ -36,4 +36,65 @@
     </xsd:complexType>
     <xsd:element name="ImportCertificateResponseMessage" type="tns:ImportCertificateResponseMessageType"/>
     
+    
+    <xsd:complexType name="ListTrustStoresRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListTrustStoresRequestMessage" type="tns:ListTrustStoresRequestMessageType" />
+    
+    <xsd:complexType name="ListTrustStoresResponseMessageType">
+         <xsd:sequence>
+            <xsd:element name="certData" type="xsd:base64Binary" xmime:expectedContentTypes="application/octet-stream" minOccurs="0" maxOccurs="unbounded"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListTrustStoresResponseMessage" type="tns:ListTrustStoresResponseMessageType"/>
+    
+    <xsd:complexType name="EditCertificateRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="editCertRequest" type="tns:EditCertificateRequestType" minOccurs="1" maxOccurs="1" />
```

Comment:
Can we just add String element "alias" directly in here instead of creating new complex type?